oci/private/util.bzl

```diff
@@ -171,7 +171,10 @@ if defined args (
     return win_launcher
 
 def _file_exists(rctx, path):
-    result = rctx.execute(["stat", path])
+    if rctx.os.name.startswith("windows"):
```

Comment:
This could probably could into a separate PR. note that we don't have windows CI ATM.